pkg/storage/mongo/mongo.go

```diff
@@ -79,6 +78,12 @@ func (m *ModuleStore) newSession(timeout time.Duration, insecure bool) (*mgo.Ses
 
 	dialInfo.Timeout = timeout
 
+	if dialInfo.Database != "" {
+		m.d = dialInfo.Database
+	} else {
+		m.d = conf.DefaultDBName
```

Comment:
@chriscoffee I don't think we need to set it manually b/c envconfig will do it automatically, just an FYI. But this doesn't hurt anything so nothing to change here 😄